cmd/hoarder/main.go

```diff
@@ -54,6 +56,29 @@ func main() {
 		os.Exit(1)
 	}
 
+	if *getStatus {
+		rpc, err := rpc.Dial("unix", config.RPCSocketPath)
+		if err != nil {
+			logger.Printf("Unable to open RPC socket file '%s': '%s'", config.RPCSocketPath, err)
+			os.Exit(1)
+		}
+		defer rpc.Close()
```

Comment:
Fun fact... this defer will never be executed. `os.Exit` Does not allow the execution of deferred commands.